modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/util/PowwowUtil.java

```diff
@@ -291,10 +290,6 @@ public static PowwowSubscriptionSender getPowwowSubscriptionSender(
 			"[$MEETING_JOIN_BY_PHONE_ACCESS_CODE$]",
 			PowwowServiceProviderUtil.getJoinByPhoneAccessCode(powwowMeetingId),
 			"[$MEETING_JOIN_BY_PHONE_ACCESS_CODE_LABEL$]",
-			LanguageUtil.get(
```

Comment:
should we retain this text? or maybe `PowwowServiceProviderUtil.getJoinByPhoneAccessCodeLabel()`? 
it's a kind of template processing, double check to make sure the template still works.

Reply:
My mistake, don't need to remove this label